superset/models/user_attributes.py

```diff
@@ -39,6 +41,22 @@ class UserAttribute(Model, AuditMixinNullable):
     user = relationship(
         security_manager.user_model, backref="extra_attributes", foreign_keys=[user_id]
     )
-
     welcome_dashboard_id = Column(Integer, ForeignKey("dashboards.id"))
     welcome_dashboard = relationship("Dashboard")
+    avatar_url = Column(String(100))
+
+
+class OwnershipMixin:  # pylint: disable=too-few-public-methods
```

Comment:
I meant to add a note about this, it's likely to be useful in the future if/when we decide to incorporate more UserAttributes. Thought it may come handy. Maybe I add a `NOTE:`?